inst/schema/SurveyFilters.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "sex": {
+      "type": "array",
+      "items": { "type": "string" }
```

Comment:
Will sex not always be "female", "male", "both"?

Reply:
I think we can probably enforce that, so yes, in that case, these could be hardcoded. (?)